services/cd-service/pkg/service/release_train_prognosis.go

```diff
@@ -42,6 +42,7 @@ func (s *ReleaseTrainPrognosisServer) GetReleaseTrainPrognosis(ctx context.Conte
 		WriteCommitData:       false,
 		Repo:                  s.Repository,
 		TransformerEslVersion: 0,
+		TargetType:            api.ReleaseTrainRequest_UNKNOWN.String(),
```

Comment:
Either add `TargetType` to the type `api.ReleaseTrainRequest` - or create a follow-up story to handle that.